stormdemo/source/c/build.bat

```diff
@@ -30,7 +30,7 @@ set LIBS_PATH=libs
 set KAA_LIB_PATH=%LIBS_PATH%\kaa
 set KAA_C_LIB_HEADER_PATH=%KAA_LIB_PATH%\src
 set KAA_CPP_LIB_HEADER_PATH=%KAA_LIB_PATH%\kaa
-set KAA_SDK_TAR="kaa-client*.tar.gz"
+set KAA_SDK_TAR="kaa-c*.tar.gz"
```

Comment:
If I don't mistake, as for Kaa 0.8.0 C SDK doesn't support Windows out of the box.
So how do you think. maybe we can just remove this bat-script?